core/jobs/transforms/validation/base_validation.py

```diff
@@ -380,17 +380,14 @@ class BaseValidateCommitCmdsSchema(beam.DoFn, Generic[ModelInstanceType]):  # ty
     def _get_change_domain_class(
         self, unused_item: ModelInstanceType
     ) -> Type[change_domain.BaseChange]:
```

Comment:
Yes, sort of. This is an abstract base class so the subclasses need to implement this return type.

I'm not really sure what to do with this case in general yet, TBH...we have several throughout the codebase. I figured it's fine to punt it since it doesn't really cause any issues.